docs/tutorials/consume-cloud-services.md

```diff
@@ -25,11 +25,11 @@ First, visit [VelaUX (KubeVela Dashboard)](../install#3-install-velaux), switch
 - provider-alibaba
```

Comment:
These should also be updated:)

Reply:
should all the provider-xxx be replaced as terraform-xxx? or just provider-alibaba?